support/ebpf/frametypes.h

```diff
@@ -9,39 +9,39 @@
 
 // Defines the bit mask that, when ORed with it, turn any of the below
 // frame types into an error frame.
-#define FRAME_MARKER_ERROR_BIT     0x80
+#define FRAME_MARKER_ERROR_BIT 0x80
```

Comment:
Can we restore the alignment of numbers?
Same at many other places.

Reply:
Added `AlignConsecutiveMacros` to the rule set - see https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/310/commits. Hope this satisfies this request.